molecule/with_volumes/molecule.yml

```diff
@@ -0,0 +1,48 @@
+dependency:
+  name: galaxy
+driver:
+  name: openstack
+lint:
+  name: yamllint
+  options:
+    config-file: tests/yamllint.yml
+platforms:
+  - name: fed_test1
+provisioner:
+  name: ansible
```

Comment:
I'm curious, at this point, if this might be a good place to use the delegated provisioner. But I think that, as well as swapping the Jenkinsfile over to using the oasis pipeline, is probably a topic for another ticket.

Reply:
We're not *really* provisioning anything in the create.yml playbook. So I don't really know about the need to change provisioners.

Changing to use the Jenkinsfile isn't a trivial task, because of name collisions between the two scenarios. So it will require those to be overhauled to leverage the run_hash technique or something similar.

Reply:
#9